contracts/extensions/sequential-governance/CoreGovernance.sol

```diff
@@ -165,10 +161,11 @@ abstract contract CoreGovernance is SignatureConsumer, VoteStatusConsumer, Chain
     uint256[] memory _values,
     bytes[] memory _calldatas,
     uint256[] memory _gasAmounts,
-    address _roninTrustedOrganizationContract,
-    address _gatewayContract,
-    address _creator
-  ) internal virtual returns (uint256 _round) {
+    // addressPack[0] _roninTrustedOrganizationContract,
+    // addressPack[1] _gatewayContract,
+    // addressPack[2] _creator,
+    address[] memory addressPack
```

Comment:
Restrict the array length
```suggestion
    address[3] memory addressPack
```

Reply:
I guess we can avoid using the address pack here by removing unnecessary variables.